tests/foreman/ui/test_domain.py

```diff
@@ -123,6 +123,7 @@ def test_positive_remove_parameter(session, valid_domain_name):
         assert param_name not in [param['name'] for param in params]
 
 
+@pytest.mark.e2e
```

Comment:
We aren't considering UI tests for e2e, just API and CLI

Reply:
I didn't know about that, but there is a bunch already marked and I don't have good API and CLI alternative

Reply:
Maybe we can mark whatever we want to be e2e and if the decision is strictly not to include UI tests, that can be done on pipeline level?